pkg/kubectl/clusterrolebinding.go

```diff
@@ -119,14 +119,14 @@ func (s ClusterRoleBindingGeneratorV1) StructuredGenerate() (runtime.Object, err
 	for _, user := range s.Users {
 		clusterRoleBinding.Subjects = append(clusterRoleBinding.Subjects, rbac.Subject{
 			Kind:       rbac.UserKind,
-			APIVersion: "rbac/v1alpha1",
+			APIVersion: "rbac/v1beta1",
```

Comment:
I forgot about this.  Can you fully qualify it to `rbac.authorization.k8s.io/v1beta1` and open an issues about writing a defaulter that conditionally updates it?